Upstream-Status: Pending

This pach fixes undefined symbol errors when linking against the static
libcrypto library without an explicit -lz linker flag.
It partially reverts [1] where the static library linking was introduced.

[1]: https://github.com/textmate/textmate/commit/5bbcf424874917d4f021a3f33379c6b530dae0f3

--- Frameworks/license/target.orig	2016-01-24 17:03:26.000000000 +0100
+++ Frameworks/license/target	2016-01-24 17:03:43.000000000 +0100
@@ -3,5 +3,5 @@
 LINK        += text cf
 CXX_FLAGS   += -I"$libressl_prefix/include"
 LN_FLAGS    += -Wl,-U,__Z15revoked_serialsv
-LN_FLAGS    += "$libressl_prefix"/lib/libcrypto.a
+LIBS        += crypto
 FRAMEWORKS   = Security
